src/providers/DataTypes.ts

```diff
@@ -1,6 +1,15 @@
 import { User } from 'src/types/User';
+import { ServerOrganization } from '../types/Organization';
+import React from 'react';
 
 export type ProvidedUserData = {
   user?: User;
   reloadUser: () => void;
 };
+
+export type ProvidedOrganizationData = {
+  selectedOrganization: ServerOrganization;
+  setSelectedOrganization: React.Dispatch<React.SetStateAction<ServerOrganization | undefined>>;
```

Comment:
my 2cents.. i know we have been using this pattern of exposing setState from a parent.. that's usually an internal detail in parent.
would be cleaner to maybe:
setSelectedOrganization: (org?: ServerOrganization) => void;
and in parent:
setSelectedOrganization={org => setSelectOrganziation(org)}

also for another time.. thoughts on this just being
organization: 
setOrganization: 
...
in the provider (the selected is implied..) we could just have
const { organization } = useOrganization();
across the board.

just a question. this is great for now.